src/parser/access-identifier.js

```diff
@@ -0,0 +1,13 @@
+//@flow
+import Syntax from "../Syntax";
+import type Context from "./context";
+import type { Node } from "../flow/types";
+
+export function accessIdentifier(ctx: Context): Node {
+  const node = ctx.startNode();
+  // We're removing first character which is access dot operator
+  node.value = ctx.token.value.substring(1, ctx.token.value.length);
```

Comment:
* Second argument to `substring` is unnecessary if you need the _rest of the string_ like here
* The comment says we are removing the first character but https://github.com/ballercat/walt/pull/37/files#diff-94eebbd7c72a61803345c949037690d8R75 pushes the `.` as an individual token, so I'm not sure how this works.

Reply:
I've removed it between first and second commit so I don't know why this file is still present.